src/platform/packages/shared/kbn-scout/src/playwright/page_objects/painless_lab_page.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ScoutPage } from '..';
+
+const DEFAULT_LOADING_TIMEOUT = 10_000;
+
+export class PainlessLab {
+  public outputValueElement: Locator;
+  public requestFlyoutHeader: Locator;
+
+  constructor(private readonly page: ScoutPage) {
+    this.outputValueElement = this.page.testSubj.locator('painlessTabs');
+    this.requestFlyoutHeader = this.page.testSubj.locator('painlessLabRequestFlyoutHeader');
+  }
+
+  async goto() {
+    this.page.gotoApp('dev_tools', { hash: 'painless_lab' });
+  }
```

Comment:
async operation requires `await` or `return` statement assuming it will be called in test with `await`

```suggestion
  async goto() {
    return this.page.gotoApp('dev_tools', { hash: 'painless_lab' });
  }
```